test/core/002.phpt

```diff
@@ -50,14 +50,14 @@ var_dump($r->bar);
 ===DONE===
 --EXPECTF--
 ReflectionMethodEx::__construct
-%unicode|string%(26) "ReflectionFunctionAbstract"
-%unicode|string%(7) "getName"
-%unicode|string%(3) "xyz"
+string(26) "ReflectionFunctionAbstract"
```

Comment:
Why changing core `phpt` tests here? They're not strictly relevant, can this be done in a separate PR please if they need updating (which they probably do, to be fair)...